inspire_schemas/records/elements/json_reference/citeable.yml

```
@@ -0,0 +1,17 @@
+{
+    "additionalProperties": false,
+    "properties": {
+        "$ref": {
+            "description": "hep/data record",
+            "format": "uri",
+            "minLength": 1,
+            "pattern": ".*/api/literature|data/\\d+$",
```

I think you need a parenthesis here
```suggestion
            "pattern": ".*/api/(literature|data)/\\d+$",
```